format not a string literal and no format arguments

missing module_str in call

Upstream-Status: Pending
Signed-off-by: Armin Kuster <akuster808@gmail.com>

../../../git/enforcer/src/keystate/keystate_ds.c:192:7: error: format not a string literal and no format arguments [-Werror=format-security]
|   192 |       ods_log_error_and_printf(sockfd, "Failed to run %s", cp_ds);
|       |       ^~~~~~~~~~~~~~~~~~~~~~~~


Index: git/enforcer/src/keystate/keystate_ds.c
===================================================================
--- git.orig/enforcer/src/keystate/keystate_ds.c
+++ git/enforcer/src/keystate/keystate_ds.c
@@ -189,7 +189,7 @@ exec_dnskey_by_id(int sockfd, struct dbw
 						status = 0;
 					}
 					else {
-						ods_log_error_and_printf(sockfd, "Failed to run %s", cp_ds);
+						ods_log_error_and_printf(sockfd, module_str, "Failed to run %s", cp_ds);
                                                 status = 7;
 					}
 				}
